cmake/ElektraCache.cmake

```diff
@@ -112,7 +112,7 @@ endif ()
 
 set (
 	GTEST_ROOT
-	""
+	"$ENV{GTEST_ROOT}"
```

Comment:
Is it recommended to use GTEST_ROOT from the environment per default? We never had it that way. If it is recommended, we need to update our documentation (COMPILE.md and release notes, maintainer section)

Reply:
It is an easy way to get the setting passed from the Docker Images to cmake without having to maintain it in the Jenkinsfile as well. if -DGTEST_ROOT is provided it should override whatever is in GTEST_ROOT env.

I would not emphasise it in the documentationt because it probably has no use outside of the build system.

Reply:
It will be used by everyone who has GTEST_ROOT set for whatever reason. Not mentioning it does not help there.

https://cmake.org/cmake/help/v3.0/module/FindGTest.html explicitly says: "as a CMake or environment variable". So you actually restored expected behavior (for people that know FindGTest).

It must be documented, however, because it changes the behavior of the build process.